rakelib/default_plugins.rb

```diff
@@ -119,6 +119,8 @@ module RakeLib
       logstash-input-stdin
       logstash-input-tcp
       logstash-output-stdout
+      logstash-output-null
+      logstash-codec-rubydebug
```

Comment:
@ph interesting, I removed the rubydebug and test hangs :-1:, might this be due to using the custom pipeline as in the spec_helper?